src/main/steps/applicant1Sequence.ts

```diff
@@ -468,7 +468,7 @@ export const applicant1PostSubmissionSequence: Step[] = [
   },
   {
     url: WITHDRAWING_YOUR_APPLICATION,
-    getNextStep: () => HOME_URL,
+    getNextStep: () => CONTINUE_WITH_YOUR_APPLICATION,
```

Comment:
have you checked with Sanchita that this is where we want to send them? This could cause a loop between those two pages

Reply:
I'll get back to you on this one, there is some comments from Isabel about this.

Reply:
Hi @lara-mpt, Yes Sanchita said its ok.